spec/fortnox/api/validators/context.rb

```diff
@@ -13,7 +13,7 @@
   end
   shared_examples_for 'invalid' do |attribute, values, error_type|
     values.each do |value|
-      context "when #{attribute} set to #{value}" do
+      context "when :#{attribute} set to #{value}" do
```

Comment:
This is only how to display the name of the attribute, not what type the attribute holds.